akka-stream/src/main/scala/akka/stream/impl/fusing/ActorGraphInterpreter.scala

```diff
@@ -322,7 +321,7 @@ final class GraphInterpreterShell(
 
   private var enqueueToShortCircuit: (Any) ⇒ Unit = _
 
-  lazy val interpreter: GraphInterpreter = new GraphInterpreter(assembly, mat, log, inHandlers, outHandlers, logics,
+  lazy val interpreter: GraphInterpreter = new GraphInterpreter(assembly, mat, log, logics, connections,
```

Comment:
First iteration of those was approved recently :-)
`SIP-20 Improved Lazy Val Initialization, Reviewer: Sébastien Doeraene (review comments)`
http://scala-lang.org/blog/2016/07/20/sip-summary-minutes.html